components/MultipleChoiceList.js

```diff
@@ -46,26 +45,29 @@ class MultipleChoiceList extends React.Component {
     var template = this.props.template;
 
     return (
-      <ul className={this.props.cssClass}>
+      <div className={this.props.rootClass}>
         {facetValues.map(facetValue => {
           return (
-            <li key={facetValue.name} onClick={this.handleClick.bind(this, facetValue.name)}>
+            <div className={this.props.itemClass} key={facetValue.name} onClick={this.handleClick.bind(this, facetValue.name)}>
               <Template data={facetValue} template={template} />
-            </li>
+            </div>
           );
         })}
-      </ul>
+      </div>
```

Comment:
Ok I found a way to still use ul li and it does not looks weird so let's use them :)